pkg/libvirttools/virtualization.c

```diff
@@ -20,16 +20,64 @@ limitations under the License.
 #include "virtualization.h"
```

Comment:
To sum it up, the functions in C I wrote have the following amount of libvirt calls inside:

* `defineDomain` - 2
* `createDomain` - 3
* `stopDomain` - 3
* `destroyAndUndefineDomain` - 4

It's a matter of agreeing how many libvirt calls we should consider as the necessity of making a wrapper in C. I have no strong feelings, to feel free to propose something. ;)